client/components/credit-card/docs/example.jsx

```diff
@@ -0,0 +1,52 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CreditCard from '..';
+
+class CreditCards extends Component {
+	state = {
+		cards: [
+			{
+				lastDigits: '4242',
+				cardType: 'visa',
+				name: 'James Smith',
+				expiry: '05/23',
+			},
+			{
+				lastDigits: '4444',
+				cardType: 'mastercard',
+				name: 'Jean Davis',
+				expiry: '01/21',
+			},
+		],
+	};
+
+	render() {
+		return (
+			<Fragment>
+				{ this.state.cards.map( ( card, i ) => (
+					<CreditCard key={ `${ card.lastDigits }_${ i }` } card={ card } selected={ 0 === i } />
+				) ) }
+				<CreditCard>
+					<div style={ { padding: 20 } }>
+						{ this.props.translate( 'Add another credit card...' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Add another credit card' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).